lib/constable/models/user.ex

```diff
@@ -71,9 +71,10 @@ defmodule Constable.User do
   defp require_permitted_email_domain(changeset) do
     changeset
     |> validate_change(:email, fn :email, value ->
-      case String.split(value, "@") do
-        [_, @permitted_email_domain] -> []
-        _ -> [email: "must be a member of #{@permitted_email_domain}"]
+      if String.split(value, "@") == permitted_email_domain() do
```

Comment:
Is this change equivalent to what we had before? And is it working in this current form?  

It seems to me like this equality will never evaluate to `true` because `String.split/2` will return a list of elements, but `permitted_email_domain()` is a string, right? So for example: 

```elixir
String.split("random@example.com", "@")
# => ["random", "example.com"]

String.split("someotherstring", "@")
# => ["someotherstring"]
```

In both of the cases above, you'd have: 

```elixir
## first example
if ["random", "example.com"] == "thoughtbot.com" do

## second example
if ["someotherstring"] == "thoughtbot.com"
```

Can you confirm that this is working as expected?

Reply:
it must not have been working as expected, though I did test it out! I'm gonna push up a change for this, which I've tested more thoroughly.